unittests/tools/test_edgescan_parser.py

```diff
@@ -0,0 +1,53 @@
+import json
+
+from django.test import TestCase
+
+from dojo.tools.edgescan.parser import EdgescanParser, ES_SEVERITIES
+from dojo.models import Test, Finding
+
+
+class TestEdgescanParser(TestCase):
+
+    def test_parse_file_with_no_vuln_has_no_findings(self):
+        with open("unittests/scans/edgescan/no_vuln.json") as testfile:
+            parser = EdgescanParser()
+            findings = parser.get_findings(testfile, Test())
+            self.assertEqual(0, len(findings))
+
+    def test_parse_file_with_one_vuln_has_one_findings(self):
+        with open("unittests/scans/edgescan/one_vuln.json") as testfile:
+            parser = EdgescanParser()
+            findings = parser.get_findings(testfile, Test())
+            self.assertEqual(1, len(findings))
+            finding = findings[0]
+            testfile.seek(0)
+            payload = json.loads(testfile.read())[0]
```

Comment:
This is not a good way to test. Please use fixed values.

Reply:
Fixed in 4977b14.